format-common/src/main/java/io/cdap/plugin/format/input/PathTrackingInputFormatProvider.java

```diff
@@ -38,6 +38,10 @@ protected PathTrackingInputFormatProvider(T conf) {
     this.conf = conf;
   }
 
+  public PathTrackingInputFormatProvider(String pathField) {
```

Comment:
this isn't needed

Reply:
for unit tests, you need to add a constructor to the plugin config classes (PathTrackingConfig, etc), not this class.